### PR TITLE
XSIAM ootb playbooks fix playbook inputs

### PR DESCRIPTION
**Pull request by efelmandar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4036

## Description
Fixed issues with missing/unused playbook inputs

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/CommonPlaybooks/Playbooks/playbook-Eradication_Plan.yml [hunk 1]
```diff
@@ -187,12 +187,6 @@ tasks:
             value:
               simple: "False"
           ignorecase: true
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Eradication_Plan.yml:189]: Is the operator deletion was on purpose?

File: Packs/Core/Playbooks/playbook-AWS_IAM_User_Access_Investigation.yml [hunk 2]
```diff
@@ -403,8 +382,7 @@ tasks:
           root: alert
           accessor: username
       query:
-        simple: (username:${inputs.awsUser} or hostip:${inputs.IP}) and alertsource:${alert.sourceBrand}
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-AWS_IAM_User_Access_Investigation.yml:406]: as discussed, lets remove the username from the query and remove the [0]

File: Packs/Core/Playbooks/playbook-T1036_-_Masquerading.yml [hunk 3]
```diff
@@ -572,7 +569,7 @@ tasks:
           root: alert
           accessor: username
       query:
-        simple: (initiatorsha256:${inputs.initiatorsha256} or hostip:${inputs.IP} or username:${inputs.User} or url:${inputs.URL}) and alertsource:${alert.sourceBrand} and alertname:${alert.name}
+        simple: (initiatorsha256:${alert.initiatorsha256.[0]} or hostip:${alert.hostip.[0]} or username:${alert.username.[0]} or url:${alert.url.[0]}) and alertsource:${alert.sourceBrand} and alertname:${alert.name}
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-T1036_-_Masquerading.yml:572]: lets fix the query here as well

File: Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml [hunk 4]
```diff
@@ -839,9 +839,6 @@ tasks:
         simple: "False"
       FileRemediation:
         simple: Quarantine
-      HostAutoContainment:
-        complex:
-          root: inputs.PreHostContainment
       IAMUserDomain:
         simple: '@demisto.com'
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml:843]: lets remove this default value

File: Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml [hunk 5]
```diff
@@ -930,8 +927,6 @@ tasks:
             - operator: uniq
         FileRemediation:
           simple: Quarantine
-        HostAutoContainment:
-          simple: "false"
         IAMUserDomain:
           simple: '@demisto.com'
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml:931]: here as well

File: Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml [hunk 6]
```diff
@@ -1103,9 +1095,6 @@ tasks:
         FileRemediation:
           complex:
             root: inputs.FileRemediation
-        HostAutoContainment:
-          complex:
-            root: inputs.HostAutoContainment
         IAMUserDomain:
           simple: '@demisto.com'
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml:1099]: here as well

File: Packs/CommonPlaybooks/Playbooks/playbook-Eradication_Plan.yml (hunk 1 above)
Comment (efelmandar) [reply Packs/CommonPlaybooks/Playbooks/playbook-Eradication_Plan.yml:189]: Yes, it contained the non existing playbook input and is not needed for this condition so it was removed

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**